modules/express/test/unit/clientRoutes/signPayload.ts

```diff
@@ -69,3 +69,158 @@ describe('Sign an arbitrary payload with trading account key', function () {
     await handleV2OFCSignPayload(req).should.be.resolvedWith(expectedResponse);
   });
 });
+
+describe('Sign an arbitrary payload with trading account key read from the local file system (external signing mode)', () => {
+  let bitgo: BitGo;
+
+  const walletId = '61f039aad587c2000745c687373e0fa9';
+  const walletPassword = 'wDX058%c4plL1@pP';
+  const secret =
+    'xprv9s21ZrQH143K3EuPWCBuqnWxydaQV6et9htQige4EswvcHKEzNmkVmwTwKoadyHzJYppuADB7Us7AbaNLToNvoFoSxuWqndQRYtnNy5DUY2';
+  const validPrv =
+  '{"61f039aad587c2000745c687373e0fa9":"{\\"iv\\":\\"+1u1Y9cvsYuRMeyH2slnXQ==\\",\\"v\\":1,\\"iter\\":10000,\\"ks\\":256,\\"ts\\":64,\\"mode\\":\\"ccm\\",\\"adata\\":\\"\\",\\"cipher\\":\\"aes\\",\\"salt\\":\\"54kOXTqJ9mc=\\",\\"ct\\":\\"JF5wQ82wa1dYyFxFlbHCvK4a+A6MTHdhOqc5uXsz2icWhkY2Lin/3Ab8ZwvwDaR1JYKmC/g1gXIGwVZEOl1M/bRHY420h7sDtmTS6Ebse5NWbF0ItfUJlk6HVATGa+C6mkbaVxJ4kQW/ehnT3riqzU069ATPz8E=\\"}"}';
+
+  const payload = {
+    this: {
+      is: {
+        a: 'payload',
+      },
+    },
+  };
+
+  before(() => {
+    bitgo = new BitGo({ env: 'test' });
+  });
+
+  it('should return a signed payload', async () => {
+    const stubbedSignature = Buffer.from('mysign');
+    const readFileStub = sinon.stub(fs.promises, 'readFile').resolves(validPrv);
+    const envStub = sinon.stub(process, 'env')
+      .value({ WALLET_61f039aad587c2000745c687373e0fa9_PASSPHRASE: walletPassword });
+    
+    const signMessageStub = sinon
+      .stub(Coin.Ofc.prototype, 'signMessage')
+      .resolves(stubbedSignature);
+
+    const stubbedSigHex = stubbedSignature.toString('hex');
+
+    const expectedResponse = {
+      payload: JSON.stringify(payload),
+      signature: stubbedSigHex,
+    };
+
+    const req = {
+      bitgo,
+      body: {
+        walletId,
+        payload,
+      },
+      config: {
+        signerFileSystemPath: 'signerFileSystemPath',
+      },
+    } as unknown as Request;
+
+    await handleV2OFCSignPayloadInExtSigningMode(req).should.be.resolvedWith(expectedResponse);
+    readFileStub.should.be.calledOnceWith('signerFileSystemPath');
+    signMessageStub.should.be.calledOnceWith(
+      sinon.match({
+        prv: secret,
+      })
+    );
+    readFileStub.restore();
+    envStub.restore();
+  });
+
+  describe('with invalid setup', () => {
+    const invalidPrv = '{"61f039aad587c2000745c687373e0fa9":"invalid"}';
+  
+    it('with missing wallet passphrase in env, should throw error', async () => {
+      const req = {
+        bitgo,
+        body: {
+          walletId,
+          payload,
+        },
+      } as unknown as Request;
+
+      try {
+        await handleV2OFCSignPayloadInExtSigningMode(req);
+      } catch (err) {
+        err.message.should.containEql('Could not find wallet passphrase');
+      }
```

Comment:
This test actually has a subtle flaw. If the function never throws an error, then this tests would still pass. 
One way to fix it is to manually throw in the try block.

Or we can use `should.js` to do this for us.


```suggestion
     await handleV2OFCSignPayloadInExtSigningMode(req).rejectedWith(`Could not find wallet passphrase`
```

Same comment to all the other non happy path tests.

Reply:
updated, thanks!